src/App/components/Section/components/LocationBlock/components/TimeInfo/TimeInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useSelector } from 'react-redux';
+
+import useLocations from '../../../../../../hooks/useLocations';
+import useTimeInfo from '../../../../../../hooks/useTimeInfo';
+import { IInitialState } from '../../../../../../redux/types';
+import { ITimeState } from '../../LocationBlock.types';
+
+import style from './TimeInfo.module.scss';
+import { ITimeInfoProps } from './TimeInfo.types';
+
+const TimeInfo: React.FC<ITimeInfoProps> = ({ location }) => {
+  const [time, setTime] = useState<ITimeState>({
+    hours: '',
+    minutes: '',
+    day: undefined,
+    offset: undefined,
+    suffix: '',
+    timezone: ''
+  });
+
+  const timeInfo = useTimeInfo(location);
+  const { locations } = useLocations();
+
+  const { counter, planningMode } = useSelector((state: IInitialState) => state);
+  const { showDate, showTimezone, timeFormat } = useSelector(
+    (state: IInitialState) => state.settings
+  );
+  const { userLocation } = useSelector((state: IInitialState) => state.locations);
+
+  useEffect(() => {
+    setTime(timeInfo);
+    // don't need as a dependency timeInfo
+    // eslint-disable-next-line
```

Comment:
please double check do ewe still need this comment here

Reply:
ESlint configuration is needed